src/Apps/Order/Components/AddressModal.tsx

```diff
@@ -112,7 +112,11 @@ export const AddressModal: React.FC<Props> = ({
       {show && (
         <ModalDialog
           title={title}
-          onClose={handleModalClose}
+          onClose={() => {
+            // TODO: tracking for modal close
+
+            handleModalClose()
+          }}
```

Comment:
**question:** Do we want to track how often users cancel out of this modal?

Reply:
there's some chatter on slack about this and @daytavares agrees that we should make another event to handle tracking the modal closing action.

Reply:
yup I think I have what you suggest covered here... not tracking a modal close on button press but tracking exactly which buttons were pressed